torch/_jit_internal.py

```diff
@@ -485,110 +486,43 @@ def _get_overloaded_methods(method, mod_class):
         raise Exception("Overloads are not useable when a module is redeclared within the same file: " + str(method))
     return overloads
 
-try:
-    import typing
-    from typing import Tuple, List, Dict, Optional
```

Comment:
Can you leave the imports in so that any other libraries depending on torch.jit.annotations don't fail now ? These imports are used in torchvision.

Reply:
sure